hack/addons/update-cluster-autoscaler.sh

```diff
@@ -21,7 +21,16 @@ readonly FILE_NAME="cluster-autoscaler.yaml"
 
 readonly KUSTOMIZE_BASE_DIR="${SCRIPT_DIR}/kustomize/cluster-autoscaler/"
 envsubst -no-unset <"${KUSTOMIZE_BASE_DIR}/kustomization.yaml.tmpl" >"${ASSETS_DIR}/kustomization.yaml"
-cp "${KUSTOMIZE_BASE_DIR}"/*.yaml "${ASSETS_DIR}"
+cat <<EOF >"${ASSETS_DIR}/gomplate-context.yaml"
+Cluster:
+  Name: tmpl-clustername-tmpl
+  Namespace: tmpl-clusternamespace-tmpl
+  Annotations:
+    caren.nutanix.com/cluster-uuid: tmpl-clusteruuid-tmpl
+EOF
+gomplate -f "${GIT_REPO_ROOT}/charts/cluster-api-runtime-extensions-nutanix/addons/cluster-autoscaler/values-template.yaml" \
+  --context .="${ASSETS_DIR}/gomplate-context.yaml" \
+  >"${ASSETS_DIR}/helm-values.yaml"
```

Comment:
its not clear that `helm-values.yaml` is generated. 

maybe we should include it in `.gitignore` so that people don't modify it

Reply:
In this case it's written to a temp directory which is deleted when this script exits. It is never in the repo or working copy.